src/deploy/campaign/environments.ts

```diff
@@ -172,16 +185,16 @@ export const validateEnv = (
 
   // Mainnet
   if (envLevel === "prod") {
-    requires(process.env.MOCK_MEOW_TOKEN === "false", NO_MOCK_PROD_ERR);
-    requires(process.env.STAKING_TOKEN_ADDRESS === MeowMainnet.address, STAKING_TOKEN_ERR);
     requires(!process.env.MONGO_DB_URI.includes("localhost"), MONGO_URI_ERR);
+    requires(!process.env.MOCK_Z_TOKEN, NO_MOCK_PROD_ERR);
+    requires(process.env.STAKING_TOKEN_ADDRESS === MeowMainnet.address, STAKING_TOKEN_ERR);
```

Comment:
MeowMainnet.address does not matter anymore. you are using a different token here now